packages/wallet/src/redux/state.ts

```diff
@@ -56,6 +56,8 @@ export interface ChannelSubscriptions {
 }
 export interface WaitForLogin extends SharedData {
   type: typeof WAIT_FOR_LOGIN;
+  address?: string;
```

Comment:
I'm wondering if it's better not to store the address, which is a pure function of the private key.

Reply:
Going to leave this as is for now to get the PR in, but perhaps something we can consider in the future?